src/opensles/OutputMixerOpenSLES.cpp

```diff
@@ -0,0 +1,74 @@
+/*
+ * Copyright 2017 The Android Open Source Project
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *      http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+
+#include "common/OboeDebug.h"
+#include "EngineOpenSLES.h"
+#include "OpenSLESUtilities.h"
+#include "OutputMixerOpenSLES.h"
+
+using namespace oboe;
+
+OutputMixerOpenSL *OutputMixerOpenSL::getInstance() {
+    static OutputMixerOpenSL sInstance;
+    return &sInstance;
```

Comment:
Same note about returning the reference.

Reply:
Done